CHANGELOG.md

```diff
@@ -3,7 +3,7 @@ All notable changes to magpylib are documented here.
 
 # Releases
 
-## [4.2.0] - XXXXXXXXXXX
+## [4.2.0] - 2023-01-27
 - Change pyvista plotting defaults when using `show(backend='pyvista')` to fit better with other libraries. ([#551](https://github.com/magpylib/magpylib/issues/551))
 - (Re)introducing the powerful `misc.Triangle` class that can be used to compute magnetic fields of arbitrarily shaped bodies by approximating their surface with triangular facets. ([#568](https://github.com/magpylib/magpylib/issues/568))
```

Comment:
I'd put the Triangle PR first , since the pyvista PR is not so much relevant for this release

Reply:
@OrtnerMichael also please execute `pre-commit run --all` before

Reply:
I also updated some test to reach maximum coverage